docs/jsonapi-spec.md

```diff
@@ -591,6 +592,83 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
 
 If an error occurs the command will not return `data`.
 
+### findOneAndReplace Command
+
+`findOneAndReplace` replaces a single document based on the specified filter clause. 
+
+*Syntax:*
+
+```bnf
+<find-one-and-replace-command> ::= findOneAndReplace <find-one-and-replace-command-payload> 
+<find-one-and-replace-command-payload> ::= <filter-clause>? 
+                             <sort-clause>? 
+                             <replacement-clause>?
```

Comment:
Does this need to be defined too (it's a JSON Object)? Or does it already exist?

Reply:
@johnsmartco The replacement is just a json document will replace the document in database. It can be just called `replacement` instead of `replacement-clause`. May have to add that as separate  section and defined as "The replacement document."

Reply:
Thanks @tatu-at-datastax and @maheshrajamani.  Yes, I agree - thanks, will do.

Reply:
Updated - thanks